src/Libraries/CoreNodes/DateTime.cs

```diff
@@ -302,65 +313,65 @@ public static System.TimeSpan ByDateDifference(System.DateTime date1, System.Dat
         {
             return System.TimeSpan.FromMilliseconds(timeSpan.TotalMilliseconds*scaleFactor);
         }
 
         /// <summary>
-        ///     Negates a TimeSpan.
+        ///     Negates a TimeSpan
         /// </summary>
-        /// <param name="timeSpan">A TimeSpan.</param>
-        /// <returns name="timeSpan">TimeSpan</returns>
+        /// <param name="timeSpan">TimeSpan to negate</param>
+        /// <returns name="timeSpan">Negative TimeSpan</returns>
         public static System.TimeSpan Negate(System.TimeSpan timeSpan)
         {
             return timeSpan.Negate();
         }
 
         /// <summary>
-        ///     Adds two TimeSpans.
+        ///     Adds two TimeSpans
         /// </summary>
-        /// <param name="timeSpan1">A TimeSpan.</param>
-        /// <param name="timeSpan2">A TimeSpan.</param>
-        /// <returns name="timeSpan">TimeSpan</returns>
+        /// <param name="timeSpan1">TimeSpan to add</param>
+        /// <param name="timeSpan2">Other TimeSpan to add</param>
+        /// <returns name="timeSpan">Resulting TimeSpan from addition</returns>
         public static System.TimeSpan Add(System.TimeSpan timeSpan1, System.TimeSpan timeSpan2)
         {
             return timeSpan1.Add(timeSpan2);
         }
 
         /// <summary>
-        ///     Subtracts two TimeSpans.
+        ///     Subtracts two TimeSpans
         /// </summary>
-        /// <param name="timeSpan1">A TimeSpan.</param>
-        /// <param name="timeSpan2">A TimeSpan.</param>
-        /// <returns name="timeSpan">TimeSpan</returns>
+        /// <param name="timeSpan1">TimeSpan to substract from</param>
+        /// <param name="timeSpan2">TimeSpan to substract</param>
+        /// <returns name="timeSpan">Resulting TimeSpan from subtraction</returns>
         public static System.TimeSpan Subtract(System.TimeSpan timeSpan1, System.TimeSpan timeSpan2)
         {
             return timeSpan1.Subtract(timeSpan2);
         }
 
         /// <summary>
-        ///     Attempts to parse a TimeSpan from a string.
+        ///     Attempts to parse a TimeSpan from a string
         /// </summary>
-        /// <param name="str">String representation of a TimeSpan.</param>
-        /// <returns name="timeSpan">TimeSpan</returns>
-        public static System.TimeSpan FromString(string str)
+        /// <param name="string">String representation of a TimeSpan.</param>
+        /// <returns name="timeSpan">TimeSpan from parsed string</returns>
+        public static System.TimeSpan FromString(string @string)
         {
-            return System.TimeSpan.Parse(str, CultureInfo.InvariantCulture);
+            return System.TimeSpan.Parse(@string, CultureInfo.InvariantCulture);
         }
 
         /// <summary>
         ///     Extracts the individual components of a TimeSpan.
         /// </summary>
-        /// <param name="timeSpan">A TimeSpan.</param>
+        /// <param name="timeSpan">DateTime to get components from</param>
```

Comment:
This should say `TimeSpan` rather than `DateTime`.

Reply:
Addressed in 06ff62e